### PR TITLE
Included membership information

### PR DESCRIPTION
**Pull request by elletjies**
@acrall, @ErinBecker and @Talishask I started the first draft for the membership section for the welcome tip sheet. Please feel free to critique and add suggestions based on the hidden slides here - https://docs.google.com/presentation/d/1XzoCJzkPt4DXgnM-wuV43BA3vzPVgJevY1jQ6FTXJ4s/edit#slide=id.g12575c2f660_1_10

### CODE REVIEW

**Comment by maneesha** [comment]
Looks like this is a draft so I am labeling it as such.

File: pages/welcome-tip-sheet.md [hunk 1]
```diff
@@ -45,11 +45,23 @@ You can find a full list of The Carpentries activities on our [community calenda
 - Community Discussions: Topics range anywhere from teaching workshops and developing curricula to building local communities and assessing the impact of our workshops globally. Community Discussions are added to our [community calendar](/community/#community-events) and are also listed in our [Etherpad](https://pad.carpentries.org/community-discussions).
 - Regional Community Calls: Join a call with others from your region. [Call in details](https://pad.carpentries.org/community-discussions).
 
+### Memberships
+A Member Organisation is an organisation which has made a commitment to the growth and spread of The Carpentries. Members receive priority access to [Instructor Training](https://carpentries.github.io/instructor-training/) and [Centrally-Organised Workshops](https://carpentries.org/workshops/#workshop-organising) and guidance with capacity building at their organisation.
+
+- [Benefits and tiers](https://carpentries.org/member_faq/#what-are-the-standard-membership-packages-you-offer)
+- Additional Benefits: Contact the [Membership Team](emailto:membership@carpentries.org)
```
Comment (Talishask) [inline pages/welcome-tip-sheet.md]:
```suggestion
```

File: pages/welcome-tip-sheet.md [hunk 2]
```diff
@@ -45,11 +45,23 @@ You can find a full list of The Carpentries activities on our [community calenda
...
+- [Financial Suppport](https://carpentries.org/member_faq/#is-financial-support-available)
+- [Frequently Asked Questions](https://carpentries.org/member_faq/)
+- [Membership Council](https://carpentries.org/member_faq/#what-is-the-carpentries-member-council)
+- [Request an Centrally-Organised Workshop](https://amy.carpentries.org/forms/request_workshop/)
+- [Workshop Checklist for Hosts](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html)
+- [How can your organisation Become a Member?](https://carpentries.org/membership/):
```
Comment (Talishask) [inline pages/welcome-tip-sheet.md]:
```suggestion
```

File: pages/welcome-tip-sheet.md [hunk 3]
```diff
@@ -45,11 +45,23 @@ You can find a full list of The Carpentries activities on our [community calenda
 - Community Discussions: Topics range anywhere from teaching workshops and developing curricula to building local communities and assessing the impact of our workshops globally. Community Discussions are added to our [community calendar](/community/#community-events) and are also listed in our [Etherpad](https://pad.carpentries.org/community-discussions).
 - Regional Community Calls: Join a call with others from your region. [Call in details](https://pad.carpentries.org/community-discussions).
 
+### Memberships
+A Member Organisation is an organisation which has made a commitment to the growth and spread of The Carpentries. Members receive priority access to [Instructor Training](https://carpentries.github.io/instructor-training/) and [Centrally-Organised Workshops](https://carpentries.org/workshops/#workshop-organising) and guidance with capacity building at their organisation.
+
+- [Benefits and tiers](https://carpentries.org/member_faq/#what-are-the-standard-membership-packages-you-offer)
```
Comment (Talishask) [inline pages/welcome-tip-sheet.md]:
```suggestion
- [How can your organisation Become a Member?](https://carpentries.org/membership/): 
- [Benefits and tiers](https://carpentries.org/member_faq/#what-are-the-standard-membership-packages-you-offer)
```

File: pages/welcome-tip-sheet.md [hunk 4]
```diff
@@ -45,11 +45,23 @@ You can find a full list of The Carpentries activities on our [community calenda
...
+- Additional Benefits: Contact the [Membership Team](emailto:membership@carpentries.org)
+- [Financial Suppport](https://carpentries.org/member_faq/#is-financial-support-available)
+- [Frequently Asked Questions](https://carpentries.org/member_faq/)
+- [Membership Council](https://carpentries.org/member_faq/#what-is-the-carpentries-member-council)
```
Comment (Talishask) [inline pages/welcome-tip-sheet.md]:
```suggestion
```

File: pages/welcome-tip-sheet.md [hunk 5]
```diff
@@ -45,11 +45,23 @@ You can find a full list of The Carpentries activities on our [community calenda
...
+- [Request an Centrally-Organised Workshop](https://amy.carpentries.org/forms/request_workshop/)
```
Comment (Talishask) [inline pages/welcome-tip-sheet.md]:
```suggestion
- When you are ready to begin planning a workshop with The Carpentries, please let us know 2-3 months in advance by [requesting a Centrally-Organised Workshop](https://amy.carpentries.org/forms/request_workshop/)
```

File: pages/welcome-tip-sheet.md [hunk 6]
```diff
@@ -45,11 +45,23 @@ You can find a full list of The Carpentries activities on our [community calenda
...
+- [Workshop Checklist for Hosts](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html)
```
Comment (Talishask) [inline pages/welcome-tip-sheet.md]:
```suggestion
- You'll receive a registration code to share with your trainees. Trainees can register for any Instructor Training event that fits their schedule. Check out the [Instructor Training Calendar](https://carpentries.github.io/instructor-training/training_calendar/) for a full list of events.
```

Comment (Talishask) [reply pages/welcome-tip-sheet.md]: The workshops checklist seems too detailed for the member organisation checklist. Maybe we should take an idea back to WAT to share this once the WRF has been submitted.

File: pages/welcome-tip-sheet.md (hunk 2 above)
Comment (Talishask) [reply pages/welcome-tip-sheet.md]: moved up

File: pages/welcome-tip-sheet.md (hunk 1 above)
Comment (Talishask) [reply pages/welcome-tip-sheet.md]: This is an option but probably not something we need to point out on the welcome tip sheet. This links to contact the team and we already have a point for this at the end.

**Review by Talishask — COMMENTED** [review]
Thanks for drafting! I left some reorganisation and additional suggestions to make the content valid for folks interested in becoming members and also folk who are members and need onboarding (or to know what to do). Tried to keep it small.

**Review by ErinBecker — APPROVED** [review]
Thanks so much for putting this together @elletjies! I agree with @Talishask 's changes and am submitting my review as approval. 🙂

**Comment by acrall** [comment]
Waiting to finish this until after mine and @elletjies next 1:1 on 12 May so we can discuss changes to this and welcome session presentation

File: pages/welcome-tip-sheet.md [hunk 7]
```diff
@@ -45,11 +45,21 @@ You can find a full list of The Carpentries activities on our [community calenda
 - Community Discussions: Topics range anywhere from teaching workshops and developing curricula to building local communities and assessing the impact of our workshops globally. Community Discussions are added to our [community calendar](/community/#community-events) and are also listed in our [Etherpad](https://pad.carpentries.org/community-discussions).
 - Regional Community Calls: Join a call with others from your region. [Call in details](https://pad.carpentries.org/community-discussions).
 
+### Memberships
+A Member Organisation is an organisation which has made a commitment to the growth and spread of The Carpentries. Members receive priority access to [Instructor Training](https://carpentries.github.io/instructor-training/) and [Centrally-Organised Workshops](https://carpentries.org/workshops/#workshop-organising) and guidance with capacity building at their organisation.
+
+- [How can your organisation Become a Member?](https://carpentries.org/membership/)
+- [Benefits and tiers](https://carpentries.org/member_faq/#what-are-the-standard-membership-packages-you-offer)
```
Comment (acrall) [inline pages/welcome-tip-sheet.md]: To be consistent with capitalization, I would capitalize "Tiers"

File: pages/welcome-tip-sheet.md [hunk 8]
```diff
@@ -45,11 +45,21 @@ You can find a full list of The Carpentries activities on our [community calenda
...
+- [Financial Suppport](https://carpentries.org/member_faq/#is-financial-support-available)
```
Comment (acrall) [inline pages/welcome-tip-sheet.md]: Suppport should be "Support"

File: pages/welcome-tip-sheet.md [hunk 9]
```diff
@@ -45,11 +45,21 @@ You can find a full list of The Carpentries activities on our [community calenda
...
+- [Frequently Asked Questions](https://carpentries.org/member_faq/)
+- When you are ready to begin planning a workshop with The Carpentries, please let us know 2-3 months in advance by [requesting a Centrally-Organised Workshop](https://amy.carpentries.org/forms/request_workshop/)
```
Comment (acrall) [inline pages/welcome-tip-sheet.md]: I would remove this; too much detail for tip sheet

File: pages/welcome-tip-sheet.md [hunk 10]
```diff
@@ -45,11 +45,21 @@ You can find a full list of The Carpentries activities on our [community calenda
...
+- You'll receive a registration code to share with your trainees. Trainees can register for any Instructor Training event that fits their schedule. Check out the [Instructor Training Calendar](https://carpentries.github.io/instructor-training/training_calendar/) for a full list of events.
```
Comment (acrall) [inline pages/welcome-tip-sheet.md]: I would remove this; too much detail for tip sheet

File: pages/welcome-tip-sheet.md [hunk 11]
```diff
@@ -45,11 +45,21 @@ You can find a full list of The Carpentries activities on our [community calenda
...
+- [Contact the Membership Team](emailto:membership@carpentries.org)
```
Comment (acrall) [inline pages/welcome-tip-sheet.md]: We do not add email addresses for any of the other teams on the tip sheet. If we do this, we should be consistent. Perhaps we add a "Contact Us" in the additional resources.

File: pages/welcome-tip-sheet.md [hunk 12]
```diff
@@ -45,11 +45,21 @@ You can find a full list of The Carpentries activities on our [community calenda
...
+
+
 ### Become an Instructor
 Carpentries Instructors are the core of our community, organising and teaching workshops to spread data literacy and computational skills around the world. Becoming an Instructor is a great step to add inspiring people to your network and helps you to both develop and demonstrate your ability to teach effectively.
 
-- [Become an Instructor](/become-instructor/): webpage with all the details
```
Comment (acrall) [inline pages/welcome-tip-sheet.md:51]: I don't think this should be removed. This is an important link to have on the tip sheet

**Review by acrall — COMMENTED** [review]
Left several comments. Will discuss with @elletjies next week to finalize. Thanks for everyone's work on this

**Comment by maneesha** [comment]
Marking this as draft so it doesn't get merged... I was going to merge once I saw Erin's approval but now see that there is still work to be done.

**Comment by acrall** [comment]
@elletjies Let me know if you need anything else from me on this

File: pages/welcome-tip-sheet.md (hunk 11 above)
Comment (elletjies) [reply pages/welcome-tip-sheet.md]: @acrall added the following under additional resources: 

Should you have any:
general requests, please email [team@carpentries.org](mailto:team@carpentries.org)
workshop requests, please email [workshops@carpentries.org](mailto:workshops@carpentries.org)
community requests, please email [community@carpentries.org](mailto:community@carpentries.org)
membership requests, please email [membership@carpentries.org](mailto:membership@carpentries.org)

**Comment by elletjies** [comment]
@acrall made changes as per requested. Could you do a final review before we merge?

**Review by acrall — APPROVED** [review]
Looks great!

**Pull request merged**